bin/utest/time.c

```diff
@@ -0,0 +1,13 @@
+#include "utest.h"
+
+#include <stdio.h>
+#include <sys/time.h>
+
+int test_gettimeofday(void) {
+  timeval_t time;
+  gettimeofday(&time, NULL);
+
+  printf("Timeofday: %lld second and %d microsecond\n", time.tv_sec,
```

Comment:
This test doesn't test anything.
The test should run without any manual inspection.
Only when a test fails manual inspection is required.
Please rethink how are you going to test `gettimeofday`.